setup.py

```diff
@@ -18,7 +18,7 @@
     author="Yoshiki Vazquez-Baeza",
     author_email="yoshiki@ucsd.edu",
     description="Display ordination plots",
-    license="bsd",
+    license='BSD-3-Clause',
```

Comment:
@ElDeveloper 
>  why are we changing this anyways then?

There was a huge mixture of license holders and version strings across all the repos in the qiime2 organization, so just went through and made them all consistent both on the setup.py license and in the license files.